src/app/admin/requests/request-details/request-details.component.html

```diff
@@ -203,7 +225,7 @@
               {{ need.value }}
```

Comment:
If you made the changes with `export const demandTypes`
In `ngFor` you remove ` | keyvalue: enumUnsorted`, And use need as `[value]`
and also need for the title, but with translate, and add new translations to `ro.json` file
In the end you should have something like this

```html
<mat-option *ngFor="let need of needs" [value]="need">
  <mat-icon [svgIcon]="need"></mat-icon>
  {{ need | translate }}
</mat-option>
```

To use translate you will have to import `TranslateModule` in `RequestsModule`

Reply:
Done